posts/2024-09-19-Project-Goals-Sep-Update.md

```diff
@@ -0,0 +1,95 @@
+[#123904]: https://github.com/rust-lang/rust/issues/123904
+[#123748]: https://github.com/rust-lang/rust/issues/123748
+
+Overall:
+
+* [13 items](https://github.com/rust-lang/rust/issues?q=label%3AS-tracking-ready-for-edition+is%3Aclosed) are fully read ready for Rust 2024.
+* [10 items](https://github.com/rust-lang/rust/issues?q=label%3AA-edition-2024%20label%3AC-tracking-issue%20-label%3AS-tracking-ready-for-edition%2CS-tracking-impl-incomplete%20-label%3At-libs%20) are fully implemented but still require documentation.
+* [6 items](https://github.com/rust-lang/rust/issues?q=label%3AA-edition-2024%20label%3AC-tracking-issue%20-label%3AS-tracking-ready-for-edition%20label%3AS-tracking-impl-incomplete%20-label%3At-libs%20) still need implementation work.
+
+Keep in mind, there will be items that are currently tracked for the edition that will not make it.  That's OK, and we still plan to ship the edition on time and without those items.
+
+### Async Rust Parity (tracked in [#105])
+
+[#105]: https://github.com/rust-lang/rust-project-goals/issues/105
+[#129629]: https://github.com/rust-lang/rust/issues/129629
+
+We are generally on track with our marquee features: (1) Support for *async closures* is available on Nightly and the lang team arrived at a tentative consensus to keep the existing syntax (written rationale and formal decision are in progress). We issued a [call for testing](https://blog.rust-lang.org/inside-rust/2024/08/09/async-closures-call-for-testing.html) as well which has so far uncovered no issues. (2) Partial support for *Return Type Notation* is available on Nightly with the remainder under review. In addition, *dynamic dispatch for async functions* and *experimental async drop work* both made implementation progress. Async WG reorganization has made no progress. [Read the full details on the tracking issue.](https://github.com/rust-lang/rust-project-goals/issues/105#issuecomment-2361474377)
+
+
+### Stabilize features needed by Rust for Linux (tracked in [#116])
+
+[#116]: https://github.com/rust-lang/rust-project-goals/issues/116
+
+We have stabilized [extended `offset_of` syntax](https://github.com/rust-lang/rust/pull/128284) and [agreed to stabilize Pointers to Statics in Constants](https://github.com/rust-lang/rust/issues/128183). Credit to @dingxiangfei2009 for driving these forward. 💜
+
+Implementation work proceeds for [arbitrary self types v2](https://github.com/rust-lang/rust/issues/44874#issuecomment-2314739657), [derive smart](https://github.com/rust-lang/rust/pull/125048) [pointer](https://github.com/rust-lang/rust/pull/129467), and [sanitizer support](https://github.com/rust-lang/rust/pull/128348).
+
+RFL on Rust CI is implemented but still waiting on documented policy. The first breakage was detected (and fixed) in [#129416](https://github.com/rust-lang/rust/pull/129416). This is the mechanism working as intended, although it is also clear that we need to better what to do when breakage occurs.
```

Comment:
>..., although it is also clear that we need to better what to do when breakage occurs.

I think this is missing a word

Reply:
```suggestion
RFL on Rust CI is implemented but still waiting on documented policy. The first breakage was detected (and fixed) in [#129416](https://github.com/rust-lang/rust/pull/129416). This is the mechanism working as intended, although it would also be useful to better define what to do when breakage occurs.
```